code/game/mecha/mecha_defense.dm

```diff
@@ -1,10 +1,10 @@
 /obj/mecha/proc/get_armour_facing(relative_dir)
 	switch(relative_dir)
-		if(0) // BACKSTAB!
+		if(180) // BACKSTAB!
 			return facing_modifiers[MECHA_BACK_ARMOUR]
-		if(45, 90, 270, 315)
+		if(90, 270)
 			return facing_modifiers[MECHA_SIDE_ARMOUR]
-		if(225, 180, 135)
+		if(0)
 			return facing_modifiers[MECHA_FRONT_ARMOUR]
 	return 1 //always return non-0
```

Comment:
pull side armor from the switch and just `return facing_modifiers[MECHA_SIDE_ARMOUR]` instead, since 1 is the side armor value anyway.

Reply:
this might be changed depending on the mech, I don't think we have an example but it might be better to keep it in for future usages